### ISSUE TITLE
Regression on PR 31 for switch case

### ISSUE THREAD

**Issue by Rassi** [Unity-Technologies/uncrustify#36]
PR 31 introduced regressions for switch case statements, not indenting them.

https://github.com/Unity-Technologies/uncrustify/pull/31#issuecomment-339662104

Expected versus output diff:
---
```diff
diff --git "a/C:\\dev\\uncrustify\\tests\\output\\staging\\60037-UNI-29933.cs" "b/C:\\dev\\uncrustify\\tests\\results\\staging\\60037-UNI-29933.cs"
index e885605..559a9f7 100644
--- "a/C:\\dev\\uncrustify\\tests\\output\\staging\\60037-UNI-29933.cs"
+++ "b/C:\\dev\\uncrustify\\tests\\results\\staging\\60037-UNI-29933.cs"
@@ -1,27 +1,27 @@
 void Foo1()
 {
     switch (foo)
     {
         case 1:
             _bar = new Bar(x, y,
-                z, a);
+            z, a);
             break;
         case 2:
             _bar = new Bar(x, y,
-                z, a);
+            z, a);
             break;
         case 3:
             _bar = foo.bar;
             break;
         case 4:
             foo.bar = Bar.BarFunc(x, (x == y)
-                ? foo.x
-                : foo.y);
+            ? foo.x
+            : foo.y);
             break;
         case 5:
             foo.bar = Bar.BarFunc(x, (x == y)
-                ? foo.x
-                : foo.y);
+            ? foo.x
+            : foo.y);
             break;
     }
 }
```

Debug support files:
---
[60037-UNI-29933.cs](https://gist.githubusercontent.com/Rassi/f9b46206970a6151b035cdb504808a31/raw/31931f883bf2f9571263b0e8af6cc94fa0bb6be4/60037-UNI-29933.cs)
[60037-UNI-29933.cs.log](https://gist.githubusercontent.com/Rassi/f9b46206970a6151b035cdb504808a31/raw/e0c7e3c81fa38e53ca5ccd43c1757e42782a350a/60037-UNI-29933.cs.log)
[60037-UNI-29933.cs.unc](https://gist.githubusercontent.com/Rassi/f9b46206970a6151b035cdb504808a31/raw/89934aa61963bc27449ede5d99b10f1ceec7a285/60037-UNI-29933.cs.unc)
[UNI-29933.cfg](https://gist.githubusercontent.com/Rassi/f9b46206970a6151b035cdb504808a31/raw/8b3ad15dd4af95fb3ee6001ea7a9a321d91f725b/UNI-29933.cfg)
[UNI-29933.cs](https://gist.githubusercontent.com/Rassi/f9b46206970a6151b035cdb504808a31/raw/fe03c949ddac51a81987ad9f3ba838ecc027d090/UNI-29933.cs)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by spuppala-rythmos** [comment]
we checked out the code before the fix went into master and ran the test case with config mentioned above.
we still see the issue and concluded that this is not a Regression on PR 31,
hence it will be better if new issue is created for this in the upstream.

Expected versus output diff before PR-31
---
```diff
diff --git a/eop.txt b/op
index f4ac1d93..fecc74df 100644
--- a/eop.txt
+++ b/op
@@ -4,24 +4,24 @@ void Foo1()
     {
         case 1:
             _bar = new Bar(x, y,
-                z, a);
+            z, a);^M
             break;
         case 2:
             _bar = new Bar(x, y,
-                z, a);
+            z, a);^M
             break;
         case 3:
             _bar = foo.bar;
             break;
         case 4:
             foo.bar = Bar.BarFunc(x, (x == y)
-                ? foo.x
-                : foo.y);
+            ? foo.x^M
+            : foo.y);^M
             break;
         case 5:
             foo.bar = Bar.BarFunc(x, (x == y)
-                ? foo.x
-                : foo.y);
+            ? foo.x^M
+            : foo.y);^M
             break;
     }
 }
```

**Comment by Rassi** [comment]
You're right, it does produce the same diff. I looked into it, and it was missing a config option (`indent_align_assign`) which I've added and pushed to master (1b06004).

Besides the regression (no indentation), I included only 1 level of indentation in the test, although it was previously 2 levels. So the test will still produce a diff before #31, but the test will be the correct expectation.

It is related to the discussion we had on https://github.com/Unity-Technologies/uncrustify/issues/38#issuecomment-341052772.
